### PR TITLE
What Does Security Mean in ETC

### PR DESCRIPTION
**Pull request by DonaldMcIntyre**
This is an opinion post about the meaning of security in ETC.

Chinese translation done!

It will be published on Feb 15th.

### CODE REVIEW

**Review by brolal — DISMISSED** [review]

File: content/blog/2023-02-15-what-does-security-mean-in-ethereum-classic/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+This is when Ethereum Classic will shine as a shield for the world. A shield against the beast of globalization. Suddenly, everybody will want to escape the overlord.
+
+ETC is a haven of basic rights, and basic rights are represented in it by means of a fully replicated ledger with accounts, balances, assets, programs, and smart contracts, which crosses all borders and reaches everyone in the world regardless of their country, culture, ideology, beliefs, gender, race, or any other human condition.
+
+People who believe in human agency think of finality and immutability as the result of human decision making. For example, in proof of stake systems they believe that a two thirds vote by the block validators produces finality. As if people were the guarantors of last resort. This makes proof of stake only suitable to protect people's wealth and applications from what may be called unauthorized theft.
+
+However, when it comes to authorized theft; the arbitrary over taxation, confiscation, censorship, limitation, or elimination of people's Life, Liberty and Property by tyrannical law; then proof of stake advocates are either absolutely or totally in favor of such situations.
```
Comment (TheCrowbill) [inline content/blog/2023-02-15-what-does-security-mean-in-ethereum-classic/index.md:57]: Line 57: Correct punctuation; Add hyphen; Add comma; Rewrite for clarity
```suggestion
However, when it comes to authorized theft, the arbitrary over-taxation, confiscation, censorship, limitation, or elimination of people's Life, Liberty, and Property by tyrannical law, then proof of stake advocates are either absolutely compliant or totally in favor of such situations.
```

**Review by TheCrowbill — CHANGES_REQUESTED** [review]

**Review by TheCrowbill — COMMENTED** [review]
LGTM

**Review by TheCrowbill — APPROVED** [review]

**Review by gitr0n1n — APPROVED** [review]
@DonaldMcIntyre make sure your YouTube video link works. It did not work during my review.

**Comment by DonaldMcIntyre** [comment]
> @DonaldMcIntyre make sure your YouTube video link works. It did not work during my review.

Thank you. The video is in private mode. Yes, I move it to public when I publish it. Thanks!

**Pull request merged**